tools/rosbag_storage/include/rosbag/bag.h

```diff
@@ -569,7 +581,7 @@ void Bag::doWrite(std::string const& topic, ros::Time const& time, T const& msg,
 
         // Check if we want to stop this chunk
         uint32_t chunk_size = getChunkOffset();
-        logDebug("  curr_chunk_size=%d (threshold=%d)", chunk_size, chunk_threshold_);
+        CONSOLE_BRIDGE_logDebug("  curr_chunk_size=%d (threshold=%d)", chunk_size, chunk_threshold_);
```

Comment:
Unfortunately, this doesn't work on Indigo.  From what I can tell, /usr/include/console_bridge/console.h doesn't have the `CONSOLE_BRIDGE_log*` macros on Indigo.  I have an upcoming patch which I'll push here which defines them for the `rosbag/bag.h` header file.  It's not a generic solution (like having it in `console_bridge/bridge.h` would be), but I'm not sure if we want to introduce that this late into Indigo.  Let me know what you think.